packages/block-library/src/embed/edit.native.js

```diff
@@ -223,6 +223,8 @@ const EmbedEdit = ( props ) => {
 		( 'wp-embed' === type &&
 			! NOT_PREVIEWABLE_WP_EMBED_PROVIDERS.includes( providerNameSlug ) );
 
+	const bottomSheetLabel = 'wp-embed' === type ? 'WordPress' : title;
```

Comment:
Nitpick: I see we're using the string `wp-embed` now in a couple of places so I'm wondering if we could have it as a constant instead, wdyt?

Reply:
Makes sense, added it in d917d6ff8f3fb7c3ab6371b8cd031aaf111a1929 👍